docs/conf.py

```diff
@@ -131,9 +131,9 @@ def check_if_v046():
             "icon": "fab fa-stack-overflow",
         },
         {
-            "name": "Gitter",
-            "url": "https://gitter.im/fairlearn/community",
-            "icon": "fab fa-gitter",
+            "name": "Discord",
+            "url": "https://discord.gg/R22yCfgsRn",
+            "icon": "fab fa-discord",
```

Comment:
Could you also adjust the hover behavior of the `fa-discord` icon in the css? You can change that here: https://github.com/fairlearn/fairlearn/blob/main/docs/_static/css/custom.css

For this you can use the official discord color in this case (#5765f2).

Reply:
Ah, the `custom.css` has slipped my search somehow. Thanks, it's resolved now. :+1: